server/ai_session.go

```diff
@@ -204,9 +204,32 @@ func NewAISessionSelector(ctx context.Context, cap core.Capability, modelID stri
 		return nil, err
 	}
 
+	// Periodically refresh sessions for Live Video to Video in order to minimize the necessity of refreshing sessions
+	// when the AI process is started
+	if cap == core.Capability_LiveVideoToVideo {
+		startPeriodicRefresh(sel)
+	}
+
 	return sel, nil
 }
 
+func startPeriodicRefresh(sel *AISessionSelector) {
+	go func() {
+		// Refresh and 80% or tll to avoid ever getting ttl applied
+		refreshInterval := time.Duration(0.8 * float64(sel.ttl))
+		ticker := time.NewTicker(refreshInterval)
+		defer ticker.Stop()
+		for {
+			select {
+			case <-ticker.C:
+				if err := sel.Refresh(context.Background()); err != nil {
+					clog.Infof(context.Background(), "Error refreshing AISessionSelector err=%v", err)
+				}
+			}
```

Comment:
Should we try to be a good citizen and tear down the loop when the node shuts down?

Reply:
Yeah, or some chan / signal from LivepeerNode ... not a big deal though if it seems more trouble than it is worth.